src/plugins/intel_cpu/src/nodes/mkldnn_normalize_node.cpp

```diff
@@ -1356,10 +1356,10 @@ class MKLDNNNormalizeL2Node::NormalizeL2ReferenceExecutor : public MKLDNNNormali
 
                 auto quant = post_op.quantization;
 
-                float crop_low = quant.crop_low_data->shifts_[quant.crop_low_data->count_ == 1 ? 0 : index_c];
-                float crop_high = quant.crop_high_data->shifts_[quant.crop_high_data->count_ == 1 ? 0 : index_c];
-                float input_scale = quant.input_scale_data->scales_[quant.input_scale_data->count_ == 1 ? 0 : index_c];
-                float input_shift = quant.input_shift_data->shifts_[quant.input_shift_data->count_ == 1 ? 0 : index_c];
+                float crop_low = quant.data[quant.crop_low][!quant.per_channel[quant.crop_low] ? 0 : index_c];
```

Comment:
Mind-blowing syntax

Reply:
Completely agree. Moreover, there was a mistake in the indexing that was hard to catch up considering the complicated code structure. So I have changed this part. Thank you!